tests/distributed/predict.conf

```diff
@@ -0,0 +1,4 @@
+task = predict
```

Comment:
I think we can write this file on the fly in `__init__()` or `predict()` method like all other helper files.

Reply:
added in f5e9d496715cf409f5bdf38e38092e72d0764a3d